src/feature/MyArea/hooks.ts

```
@@ -29,6 +39,30 @@ export const useMyArea = () => {
         case 'trash':
           discard({ target: { id: activeCard?.id } as ICard });
           break;
+        case 'unit':
+          // Evolution handling
+          const draggedCardId = activeCard?.id as string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)